acceptance/tests/resource/zone/ticket_4840_should_create_zone_with_zpool.rb

```diff
@@ -19,6 +19,10 @@
       mountpoint => "/ztstpool/mnt",
       ensure => present,
     }
+    file { "/ztstpool":
+      ensure => directory,
+      mode => "0755",
+    }
```

Comment:
This should be ordered to prevent the following resource from being realized out of order from this one
```
    } ->
```